### PR TITLE
Added:  km code of conduct (khmer review requested)

### PR DESCRIPTION
**Pull request by vicheanath**
## What does this PR do?
Add info

### Description
Create code-of-conduct for khmer language
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
add link from README
will ask for khmer review

**Comment by chetratep** [comment]
Why the file name is `-km` but not `-kh`?
The commonly used short of Khmer is  KH.

**Comment by eshellman** [comment]
https://www.loc.gov/standards/iso639-2/php/code_list.php

**Comment by chetratep** [comment]
I see. If it follows the ISO, 3 characters (ISO 369-2) should be used instead, I think. As you can see many of the languages doesn't has (ISO 369-1) 2 characters code. Why it was decided to use 2 characters instead of 3 characters code? 
And frankly in the Khmer community, KH or KHM is commonly used. KHM is not seen very often though.

**Comment by eshellman** [comment]
3 chars only if not 2 char code

**Comment by chetratep** [comment]
Ok.

File: docs/CODE_OF_CONDUCT-km.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# ក្រមសីលធម៌របស់អ្នកចូលរួម
+
+ក្នុងនាមជាអ្នករួមចំណែក និងអ្នកថែរក្សាគម្រោងនេះ និងក្នុងគោលបំណងជំរុញសហគមន៍បើកចំហ និងស្វាគមន៍ 
+យើងសន្យាគោរពមនុស្សទាំងអស់ដែលរួមចំណែកតាមរយៈការរាយការណ៍បញ្ហា ការបង្ហោះសំណើលក្ខណៈពិសេស 
+ការធ្វើបច្ចុប្បន្នភាពឯកសារ ការដាក់សំណើរទាញ ឬបំណះ និងសកម្មភាពផ្សេងទៀត។
+
+យើងប្តេជ្ញាធ្វើឱ្យការចូលរួមនៅក្នុងគម្រោងនេះដោយគ្មានការយាយី
+បទពិសោធន៍សម្រាប់មនុស្សគ្រប់គ្នា ដោយមិនគិតពីកម្រិតនៃបទពិសោធន៍ ភេទ ភេទ
+អត្តសញ្ញាណ និងការបញ្ចេញមតិ ទំនោរផ្លូវភេទ ពិការភាព រូបរាងផ្ទាល់ខ្លួន
+ទំហំរាងកាយ ជាតិសាសន៍ ជាតិសាសន៍ អាយុ សាសនា ឬសញ្ជាតិ។
```
Comment (chetratep) [inline docs/CODE_OF_CONDUCT-km.md:10]: `ជាតិសាសន៍ ជាតិសាសន៍` This is duplicated words. I suppose the translation is from ` race, ethnicity `
race = ជាតិសាសន៍
ethnicity the closest meaning is អម្បូរ  . Other words such as ពួក Or ក្រុមមនុស្ស. However, there is no exact word in Khmer for this.  I think អម្បូរ  is more relevant here.

File: docs/CODE_OF_CONDUCT-km.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# ក្រមសីលធម៌របស់អ្នកចូលរួម
+
+ក្នុងនាមជាអ្នករួមចំណែក និងអ្នកថែរក្សាគម្រោងនេះ និងក្នុងគោលបំណងជំរុញសហគមន៍បើកចំហ និងស្វាគមន៍ 
+យើងសន្យាគោរពមនុស្សទាំងអស់ដែលរួមចំណែកតាមរយៈការរាយការណ៍បញ្ហា ការបង្ហោះសំណើលក្ខណៈពិសេស 
+ការធ្វើបច្ចុប្បន្នភាពឯកសារ ការដាក់សំណើរទាញ ឬបំណះ និងសកម្មភាពផ្សេងទៀត។
+
+យើងប្តេជ្ញាធ្វើឱ្យការចូលរួមនៅក្នុងគម្រោងនេះដោយគ្មានការយាយី
+បទពិសោធន៍សម្រាប់មនុស្សគ្រប់គ្នា ដោយមិនគិតពីកម្រិតនៃបទពិសោធន៍ ភេទ ភេទ
+អត្តសញ្ញាណ និងការបញ្ចេញមតិ ទំនោរផ្លូវភេទ ពិការភាព រូបរាងផ្ទាល់ខ្លួន
```
Comment (chetratep) [inline docs/CODE_OF_CONDUCT-km.md:9]: `ភេទ អត្តសញ្ញាណ` It should `អត្តសញ្ញាណភេទ`. 
`Gender Identity` we don't this word in Khmer. But we can use word command translation with no space to indicated that it is one word.

File: docs/CODE_OF_CONDUCT-km.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+តាមរយៈការអនុម័តក្រមសីលធម៌នេះ អ្នកថែទាំគម្រោងបានប្តេជ្ញាចិត្តចំពោះខ្លួនឯង
+ការអនុវត្តគោលការណ៍ទាំងនេះដោយយុត្តិធម៌ និងជាប់លាប់ចំពោះគ្រប់ទិដ្ឋភាពនៃការគ្រប់គ្រង
+គម្រោងនេះ។ អ្នកថែទាំគម្រោងដែលមិនអនុវត្តតាម ឬអនុវត្តក្រមនៃ
+ការប្រព្រឹត្តអាចនឹងត្រូវដកចេញជាអចិន្ត្រៃយ៍ពីក្រុមគម្រោង។
+
+ក្រមសីលធម៌នេះអនុវត្តទាំងក្នុងចន្លោះគម្រោង និងក្នុងទីសាធារណៈ
+នៅពេលដែលបុគ្គលម្នាក់តំណាងឱ្យគម្រោង ឬសហគមន៍របស់ខ្លួន។
+
+ករណីនៃការបំពាន ការយាយី ឬអាកប្បកិរិយាមិនអាចទទួលយកបានអាចកើតមាន
+រាយការណ៍ដោយទាក់ទងអ្នកថែទាំគម្រោងនៅ victorfelder នៅ gmail.com ។ ទាំងអស់។
```
Comment (chetratep) [inline docs/CODE_OF_CONDUCT-km.md:36]: Anyway, here I think it is a gmail address right? 
I saw in https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CODE_OF_CONDUCT.md
is using `victorfelder at gmail.com` as well. 
Would look betting if we just use a proper email address instead? `victorfelder@gmail.com`
It is just my idea. You may have your own reason.

**Comment by eshellman** [comment]
thank you @chetratep for review. @vicheanath please address the comments

**Comment by vicheanath** [comment]
I have modified best on review done.

**Comment by vicheanath** [comment]
> I see. If it follows the ISO, 3 characters (ISO 369-2) should be used instead, I think. As you can see many of the languages don't has (ISO 369-1) 2 characters code. Why it was decided to use 2 characters instead of 3 characters code? And frankly in the Khmer community, KH or KHM is commonly used. KHM is not seen very often though.

Yes for sure we use KH for general use as Khmer, everybody knows `KH` = Khmer 
but km is a default code for every software using i18n. we use km as a standard code, and for 3 char I think it is usually used for currency codes like `KHR`, and `USD` some think like that. i'm certify that is okay for use `km`

**Review by chetratep — APPROVED** [review]
LGMT.
Hopefully to the see the index added soon.

**Pull request merged**